en/services/opendroneid.md

```diff
@@ -3,43 +3,71 @@
+### Europe {#europe}
+
+To meet the European Commission Delegated Regulation [2019/945](https://eur-lex.europa.eu/eli/reg_del/2019/945/2020-08-09) and the Commission Implementing Regulation [2019/947](https://eur-lex.europa.eu/eli/reg_impl/2019/947/2021-08-05), ASD-STAN has developed the prEN 4709-002 Direct Remote Identification standard.
+This standard specifies broadcast methods for Remote ID (Bluetooth and Wi-Fi) that are compliant with the ASTM F3411 specification.
+See the summary [whitepaper](https://asd-stan.org/wp-content/uploads/ASD-STAN_DRI_Introduction_to_the_European_digital_RID_UAS_Standard.pdf).
+The final version is not yet published (September 2021).
+An earlier draft is available [here](https://asd-stan.org/downloads/din-en-4709-0022021-02/).
+Be aware, that multiple changes were done after this draft version was made available and it should not be used as a reference for any implementations.
+
+## Broadcast methods and impact on the message design {#broadcast_methods}
+
+Four different broadcast methods are defined:
+* Bluetooth Legacy Advertising (Bluetooth 4.x)
+* Bluetooth Long Range with Extended Advertising (Bluetooth 5.x)
+* Wi-Fi Neighbor-aware Network (Wi-Fi NaN)
+* Wi-Fi Beacon (vendor specific information element in the SSID beacon frame)
+
+The broadcast method used with Bluetooth Legacy Advertising signals impose a strict size limitation on the amount of data that can be transmitted in each radio burst.
+Therefore the relevant data is divided into different categories and each category is transmitted via its own message.
+
+The ASTM Remote ID standard defines 6 such messages and an additional 7th message type for packing multiple messages together into a message pack (used when transmitting on Wi-Fi NaN, Wi-Fi Beacon or Bluetooth Long Range with Extended Advertising).
 To support easy data transfers to/from a drone ID transmitter/receiver, MAVLink messages supporting all the fields of the drone ID messages have been made available.
+See [Messages](#messages) below.
+
+## Use case examples {#use_cases}
 
 There are multiple possible use cases for the MAVLink drone ID messages:
-* A flight controller sends ID, location etc. data to an onboard Bluetooth/WiFi transmitter module.
-* An onboard Bluetooth/WiFi receiver picks up drone ID messages from surrounding aircrafts, relays this information using MAVLink drone ID messages to the flight controller, which then uses the information e.g. for Detect And Avoid (DAA) calculations.
-* A drone sends MAVLink drone ID messages via it's control link to the Ground Control Station (GCS).
+* A flight controller sends ID, location etc. data to an onboard Bluetooth/Wi-Fi transmitter module.
+* An onboard Bluetooth/Wi-Fi receiver picks up drone ID messages from surrounding aircrafts, relays this information using MAVLink drone ID messages to the flight controller, which then uses the information e.g. for Detect And Avoid (DAA) calculations.
```

Comment:
aircrafts > airfcraft. That won't stop me merging this as I assume it is copied spec terminology.

Reply:
I don't really follow? I changed it's to its and WiFi to Wi-Fi.